gematria/testing/python/ir_utils_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2024 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from absl.testing import absltest
+from gematria.testing.python import ir_utils
+
+
+class IRUtilsTests(absltest.TestCase):
+
+  def test_get_bc_from_ir(self):
+    ir_string = """
+define i32 @a() {
+  ret i32 0
+}
+"""
```

Comment:
IIRC, whitespace doesn't matter in the IR code, so this could be
```python
    ir_string = """
        define i32 @a() {
          ret i32 0
        }
        """
```

If you prefer to keep the IR code unindented, `textwrap.dedent()` can help:
```python
    ir_string = textwrap.dedent("""\
        define i32 @a() {
          ret i32 0
        }
        """)
```

Same below.

Reply:
It's needed in at least one case (`test_round_trip`). I've used `textwrap.dedent()` on all of them just to keep things clean.